core/src/main/java/bisq/core/trade/Trade.java

```diff
@@ -30,6 +30,7 @@
 import bisq.core.support.dispute.refund.RefundResultState;
 import bisq.core.support.messages.ChatMessage;
 import bisq.core.trade.protocol.ProcessModel;
+import bisq.core.trade.protocol.ProcessModelI;
```

Comment:
Same here I find adding the I to the end to mark the interface kind of confusing. It could be mistaken as the short form of `Impl`.